src/pages/apply/FormPage.js

```diff
@@ -31,274 +32,554 @@ const FormPage = () => {
-                    </span>
-                  </Control>
-                </Field>
-              </Column>
-            </Columns>
-            <Columns>
-              <Column>
-                <Field>
-                  <Label>Preferred Gender Pronoun</Label>
-                  <Control>
-                    <div className="select">
-                      <Select name="genderPronoun">
-                        <option selected disabled>
-                          select your gender pronoun
-                        </option>
-                        <option>he</option>
-                        <option>she</option>
-                        <option>they</option>
-                        <option>prefer not to say</option>
-                      </Select>
-                    </div>
-                  </Control>
-                </Field>
-              </Column>
-              <Column>
-                <Field>
-                  <Label>Birthday</Label>
-                  <Control>
-                    <Input type="date" name="birthday" />
-                  </Control>
-                </Field>
-              </Column>
-              <Column>
-                <Field>
-                  <Label>Occupation</Label>
-                  <Control>
-                    <Input type="text" name="occupation" />
-                  </Control>
-                </Field>
-              </Column>
-            </Columns>
-            <SectionHeader>Social Background</SectionHeader>
-            <legend>
-              Please list your social media handles where applicable
-            </legend>
-            <Columns>
-              <Column>
-                <Control className="has-icons-left">
-                  <Input type="text" name="twitter" />
-                  <span class="icon is-small is-left twitter">
-                    <TiSocialTwitter />
-                  </span>
-                </Control>
-              </Column>
-              <Column>
-                <Control className="has-icons-left">
-                  <Input type="text" name="instagram" />
-                  <span class="icon is-small is-left instagram">
-                    <TiSocialInstagram />
-                  </span>
+          <Formik
+            initialValues={{
+              firstName: "",
+              lastName: "",
+              email: "",
+              phoneNumber: "",
+              physicalAddress: "",
+              pronoun: "",
+              birthDay: "",
+              occupation: "",
+              twitterHandle: "",
+              instagramHandle: "",
+              facebookHandle: "",
+              linkedinHandle: "",
+              publicFigures: "",
+              mostAdmiredNGO: "",
+              preferredSocialMedia: "",
+              volunteerType: "",
+              interestedVolunteerUnits: [],
+              reasonForVolunteering: "",
+            }}
+            onSubmit={(values) => {
+              const formattedData = {
```

Comment:
We can assign a separate function to handle this object construction

So onSubmit does its work of submitting

Reply:
This will be resolved in the next life 🤗